style.css

```diff
@@ -0,0 +1,120 @@
+header {
+  position: absolute;
+  width: 423px;
+  height: 92px;
+  left: 0;
+  top: 0;
+  background: #3c3a39;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.name {
+  color: white;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-weight: bold;
+  margin-top: 45px;
+  margin-left: 20px;
+}
+
+.navi {
+  margin-left: 250px;
+  margin-top: 45px;
+}
+
+section {
+  display: flex;
```

Comment:
### OPTIONAL
_I write it as an optional but this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times) that is not mentioned in the last reviews._
- [ ] For DRY CSS you should combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.

- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/)